endpoints/cookie_sync.go

```diff
@@ -63,6 +63,7 @@ func NewCookieSyncEndpoint(
 type cookieSyncEndpoint struct {
 	chooser          usersync.Chooser
 	config           config.UserSync
+	accountConfig    config.Account
```

Comment:
This data structure is for the life of the application, but  account config should be for the life of the request. You'll want to add the account as a string in the `cookieSyncRequest` struct and use that with the `stored_requests.AccountFetcher` to lookup the account config. So it's really the AccountFetcher you want to include at this level.

There is also no automatic dependency injection in this project. You'll need to wire that up manually in router.go#New. Look for the line:

```
r.POST("/cookie_sync", endpoints.NewCookieSyncEndpoint(syncersByBidder, cfg, gdprPerms, r.MetricsEngine, pbsAnalytics, activeBidders).Handle)
```

where you'll have the `accounts` variable for the AccountFetcher in scope.